src/collection/collection.types.ts

```diff
@@ -71,5 +71,5 @@ export interface MarcomEntry {
 }
 
 export interface FragmentComponent {
-	fragment: CollectionFragment;
+	fragment: CollectionFragment | AssignmentBlock;
 }
```

Comment:
Optional: Ik zou opteren om hierboven een type aan te maken (via bv. `Pick<...>`) met enkel de gedeelde properties die momenteel van toepassing zijn. Op die manier vermijden we dat we down-the-line beginnen checken welk type er binnenkomt en opnieuw conditionals beginnen tussenknallen.

Reply:
dit bedoelde ik met een hoofd type:
```
interface BlockBase {
  item_meta: ...
  type: ....
}

interface CollectionFragment extends BlockBase {
  ...
}

interface AssignmentBlock extends BlockBase {
  ...
}
```

Dan kunnen we BlockBase in die functie gebruiken